compiler/luci/lang/include/luci/IR/CircleNodes.h

```diff
@@ -132,6 +133,7 @@
 #include "Nodes/CircleBCQGather.h"
 #include "Nodes/CircleInstanceNorm.h"
 // Virtual nodes
+#include "Nodes/CircleBidirectionalSequenceLSTMOut.h"
```

Comment:
can you please put this line after `#include "Nodes/CircleOutput.h"`?

Reply:
will address this, thanks.